app/pandas.ipynb

```diff
@@ -292,33 +368,66 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
-   "outputs": [],
    "source": [
-    "# merge"
+    "data1 = {'Name': ['John', 'Emily', 'Ryan'],\n",
+    "         'Age': [25, 28, 22]}\n",
+    "data2 = {'Name': ['Emily', 'Ryan', 'Mike'],\n",
+    "         'City': ['Los Angeles', 'Chicago', 'Houston']}\n",
+    "df1 = pd.DataFrame(data1)\n",
+    "df2 = pd.DataFrame(data2)\n",
+    "\n",
+    "merged_df = pd.merge(df1, df2, on=\"Name\", how=\"inner\")\n",
+    "print(merged_df)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-05T09:41:41.152019Z",
-     "start_time": "2023-07-05T09:41:41.137327Z"
+     "end_time": "2024-11-11T19:06:53.414875Z",
+     "start_time": "2024-11-11T19:06:53.402016Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    Name  Age         City\n",
+      "0  Emily   28  Los Angeles\n",
+      "1   Ryan   22      Chicago\n"
+     ]
+    }
+   ],
+   "execution_count": 18
   },
   {
    "cell_type": "code",
-   "execution_count": 24,
-   "outputs": [],
    "source": [
-    "# concatenate"
+    "concatenated_df = pd.concat([df1, df2], ignore_index=True)\n",
+    "print(concatenated_df)"
```

Comment:
The concatenation operation is correctly implemented, but ensure you also demonstrate the difference between concatenation and merging as described in Task 5.1.